src/lib/y2s390/dasd_actions/format.rb

```diff
@@ -0,0 +1,84 @@
+require "y2s390/dasd_actions/base"
+
+module Y2S390
+  module DasdActions
+    class FormatOff < Base
+      def run
+        selected.each { |dasd| dasd.format_wanted = false }
+        true
+      end
+    end
+
+    class FormatOn < Base
```

Comment:
The same as above: `FormatOn < Format` instead of `FomatOn < Base`

Reply:
:thinking: Or maybe not, since this action performs a different #run. Sorry, I got confused with the "analogous" DiagOn.